### PR TITLE
[READY FOR REVIEW] Japanese - WCAG 2 Overview

### PR DESCRIPTION
**Pull request by MakotoUeki**
First version of Japanese translation "WCAG 2 Overview"

* I'm a first timer. So I'm not sure if I'm doing it right...

Resolves https://github.com/w3c/wai-translations/issues/231

### CODE REVIEW

**Comment by MakotoUeki** [comment]
@remibetin @shawna-slh 
Could you please check if I'm doing it right so far? Thanks in advance for your time.

**Comment by remibetin** [comment]
Hi @MakotoUeki,

I have just updated your branch so that the file is `index.ja.md` rather than `index.md`

I will now check the content of the file and add comments if changes are needed.

**Comment by remibetin** [comment]
@MakotoUeki You can see the preview of your translation here: https://deploy-preview-284--wai-intro-wcag.netlify.app/standards-guidelines/wcag/ja

File: content/index.ja.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+
+title: "WCAG 2 概要"
+nav_title: "Web Content – WCAG 2"
+lang: ja  # Change "en" to the translated-language shortcode
+last_updated: 2024-04-04  # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+# translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
+# - name: "Makoto Ueki" # Add one -name: line for every translator
```
Comment (remibetin) [inline content/index.ja.md:11]:
```suggestion
# - name: "Makoto Ueki" # Add one -name: line for every translator
```

Please remove the hash sign and the space at the beginning of this line to make your name appear as translator.

File: content/index.ja.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+# - name: "Makoto Ueki"   # Replace Jan Doe with translator name
```
Comment (remibetin) [inline content/index.ja.md:12]: This line is redundant and can be removed

File: content/index.ja.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+
+title: "WCAG 2 概要"
+nav_title: "Web Content – WCAG 2"
```
Comment (remibetin) [inline content/index.ja.md:6]: @MakotoUeki Can you please translate "Web Content"?

File: content/index.ja.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+description: WCAG2.0、WCAG2.1、WCAG2.2を含む、国際標準のウェブコンテンツアクセシビリティガイドライン（WCAG）を紹介。WCAG文書は、ウェブコンテンツを障害がある人にとってよりアクセシブルにする方法を説明しています。
+image: /content-images/wai-intro-wcag/wcag-intro-social.jpg
+feedbackmail: wai@w3.org
+
+# In the footer below:
+# Do not change the dates
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+footer: >
+  <p><strong>最終更新日：</strong> 7 March 2024、初版公開 July 2005</p>
```
Comment (remibetin) [inline content/index.ja.md:32]: @MakotoUeki Can you please translate the dates?

File: content/index.ja.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Summary" class="full" %}
+{:/}
+
+このページでは、WCAG 2.0、WCAG 2.1、およびWCAG 2.2を含む、国際標準であるWebコンテンツ・アクセシビリティ・ガイドライン（WCAG）について紹介します。WCAG文書は、Webコンテンツを障害がある人にとってよりアクセシブルにする方法を説明しています。
+
+関連ページ [introduces WCAG 3＜英語＞](/standards-guidelines/wcag/wcag3-intro/).
+
+WCAGはアクセシビリティの入門書ではありません。入門書のようなドキュメントとしては、[Accessibility Fundamentals Overview＜英語＞](/fundamentals/) を参照してください。
```
Comment (remibetin) [inline content/index.ja.md:45]: @MakotoUeki Please translate the text in the links and do not manually add `(in English)`, even for external links.

See context in https://www.w3.org/WAI/about/translating/guides/translation-guidance/#links

File: content/index.ja.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+* [How to Meet WCAG 2 (Quick Reference)＜英語＞](https://www.w3.org/WAI/WCAG22/quickref/)
+* [WCAG 2.2 Standard＜英語＞](https://www.w3.org/TR/WCAG22/), [What's New in WCAG 2.2＜英語＞](/standards-guidelines/wcag/new-in-22/)
+* [WCAG 2.1 Standard＜英語＞](http://www.w3.org/TR/WCAG21/)
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Page Contents" class="simple" %}
```
Comment (remibetin) [inline content/index.ja.md:57]: @MakotoUeki "Page Contents" needs translation

File: content/index.ja.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+
+WCAGは、オーサリングツール アクセシビリティガイドライン（ATAG）やユーザーエージェント アクセシビリティガイドライン（UAAG）を含む一連のアクセシビリティガイドラインの1つです。[Essential Components of Web Accessibility＜英語＞](/fundamentals/components/)では、さまざまなガイドライン間の関係性について説明しています。
+
+以下に関する詳細については、[WCAG 2 FAQ](/standards-guidelines/wcag/faq/)を参照してください：
+
+-   ** [mobile accessibility＜英語＞](/standards-guidelines/wcag/faq/#mobile)**においてWCAG 2がカバーする範囲
+-   **  [documents and software＜英語＞](/standards-guidelines/wcag/faq/#wcag2ict)**へのWCAG 2の適用
+-   その他
+
+W3C アクセシビリティガイドライン3.0（旧称「Sliver」）の初期ドラフトに関する情報は、**[WCAG 3 Introduction](/standards-guidelines/wcag/wcag3-intro/)**をご覧ください。
```
Comment (remibetin) [inline content/index.ja.md:162]: @MakotoUeki "Sliver" is a typo that will be fixed very soon in the original resource: sorry for that. The name was "Silver". Would you please translate it into Japanese?

File: content/index.ja.md (hunk 5 above)
Comment (remibetin) [reply content/index.ja.md:45]: @MakotoUeki Please note this applies to all links in the translation.

File: content/index.ja.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+各ガイドラインにはテスト可能な達成基準があり、達成基準は[A、AA、AAA という3つのレベル](https://www.w3.org/WAI/WCAG22/Understanding/conformance#levels)に分かれています。
+
+達成基準は、WCAGへの「適合性」を決定するものです。つまり、WCAGに適合するためには、コンテンツが達成基準を満たしている必要があります。詳細は、[WCAGの「適合性」のセクション＜英語＞](https://www.w3.org/TR/WCAG22/#conformance)に記載されています。
+
+WCAG 2ガイドラインの簡潔なサマリーは、**[WCAG 2 at a Glance＜英語＞](/standards-guidelines/wcag/glance/)**を参照してください。
+
+### サポート資料および補足ガイダンス {#supplement}
+
+以下のリソースは、WCAGを理解して実装し、WCAGを超えてアクセシビリティを高めるのに役立ちます：
+* Quick Reference / How to Meet WCAG 2 / Checklist
```
Comment (remibetin) [inline content/index.ja.md:103]: @MakotoUeki The list items need translation.

**Comment by remibetin** [comment]
@MakotoUeki Thanks again for your contribution! I have skimmed the translation and added comments to help you complete it. Let me know when a new version is available so I can take another look.

**Comment by MakotoUeki** [comment]
@remibetin Thanks for checking my file. I'm just done with fixes and just updated the file. Could you please take another look?

**Review by remibetin — APPROVED** [review]
Thanks very much @MakotoUeki!

**Comment by shawna-slh** [comment]
Thank you @MakotoUeki and @remibetin !

**Comment by remibetin** [comment]
QA done: ✅

**Pull request merged**